pkg/runner/runner.go

```diff
@@ -176,15 +178,28 @@ func selectParameters(plan bundle.Plan) (bundle.Parameters, error) {
 	schemaParams := planSchema.ServiceInstance.Create["parameters"]
 	params := bundle.Parameters{}
 	for _, param := range plan.Parameters {
+		var inputValid = false
```

Comment:
`var inputValid bool` would be better since the zero value for booleans is false. This would be inline like the line below where we do `var paramDefault interface{}`.

Reply:
From a code readability perspective, it feels clearer to me to explicitly set `inputValid` to false here, but perhaps this is a Golang convention I'm not privy to quite yet :)